mne/io/tests/test_constants.py

```diff
@@ -36,6 +36,7 @@
                      'viewitems', 'viewkeys', 'viewvalues',  # Py2
                      )
 _tag_ignore_names = (
+    'FIFF_MNE_EPOCHS_RAW_SFREQ',
```

Comment:
You can now remove this and change the commit above to `aa49e20cff5791fbaf01d77ad4ec2e0ecb69840d`

Reply:
It looks like you did not update the commit hash above in this file, I expect `pytest mne/io/tests/test_constants.py` to fail

Reply:
Oh sorry I misunderstood what you were saying. Updating now.

Reply:
Sounds good -- please do test that line (plus other epochs-annotatinos-relevant tests) locally before pushing... it's a lot faster and lower CI load than waiting for CIs to complain.